### PR TITLE
Fixes blueprints not working on lavaland

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
You can now build/ashproof your own shelters

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
immersive

File: code/game/objects/items/blueprints.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@
 				<small>[fluffnotice]</small><hr>"
 	switch(get_area_type())
 		if(AREA_SPACE)
-			text += "<p>According to the [src.name], you are now in <b>outer space</b>.  Hold your breath.</p> \
```
Comment (duncathan) [inline code/game/objects/items/blueprints.dm]: rip good fluff text

File: code/game/objects/items/blueprints.dm [hunk 2]
```diff
@@ -112,7 +112,7 @@
 
 
 /obj/item/areaeditor/proc/get_area_type(area/A = get_area())
```
Comment (duncathan) [inline code/game/objects/items/blueprints.dm]: unrelated but this proc is a monstrosity

hell all the procs for this path are, really

Comment (KorPhaeron) [inline code/game/objects/items/blueprints.dm]: I tried not to dig too deep into this nightmare of a file when fixing this

**Pull request merged**